### ISSUE TITLE
Crossing-repair should support tied items

### ISSUE THREAD

**Issue by rpherbig** [rpherbig/dr-scripts#116]
E.g. Sheltim's bow

**Comment by rpherbig** [comment]
```
>
[crossing-repair]>get my bow
You pull at it, but the ties prevent you.  Maybe if you untie it, first?
>
[crossing-repair]>give my bow to Randal
What is it you're trying to give?
>
[crossing-repair: ]
[crossing-repair: *** ***]
[crossing-repair: ***bow not found!***]
[crossing-repair: *** ***]
```

**Comment by rpherbig** [comment]
Fixed in c139299b25a327577036dbe5795f751b21c252c1
